docs/topics/proc_show-debug-logging-openshift.adoc

```diff
@@ -46,10 +46,18 @@ $ curl $APPLICATION_ROUTE/api/greeting?name=Sarah
 ----
 
 . Return to your pod logs and notice there are debug logging messages in the logs.
+ifdef::spring-boot-logging[]
 +
 [source,bash,options="nowrap",subs="attributes+"]
 ----
 i.o.booster.service.GreetingEndpoint     : Message: Hello, Sarah!
 ---- 
+endif::spring-boot-logging[]
```

Comment:
Should we maybe have examples for the other runtimes? Not in this PR, but for the future.

Also the text above (line 23 specifically) uses `mvn fabric8:deploy` -- is this topic also used in the Node.js guide?

Reply:
I'm fairly sure that the Node guide contains a different module, specifically `proc_show-debug-logging-openshift-nodejs.adoc`  that covers this topic.

Reply:
Ah OK.